cmd/tikv-ctl/src/main.rs

```diff
@@ -971,29 +1006,25 @@ fn flashback_whole_cluster(
     println!("flashback all stores success!");
 }
 
-fn load_leaders_to_each_store(
+// Load (region_id, leader's store id) in the cluster with key ranges.
+fn load_key_range(
     pd_client: &RpcClient,
     start_key: Vec<u8>,
     end_key: Vec<u8>,
-) -> HashMap<u64, HashMap<(Vec<u8>, Vec<u8>), u64>> {
+) -> HashMap<(Vec<u8>, Vec<u8>), (u64, u64)> {
```

Comment:
Why not just make it return `Vec<metapb::Region>` here? It contains all info we need and does not need to introduce a map with such a redundant structure.

Reply:
- This map is as (key_range, info) to make sure that the unit is keyRange, if we don't map it here, we also need to construct a mapping inside the function.
- Since I only need store_id and region_id, I want to minimize redundant info when the number of regions is large.